src/index.html

```diff
@@ -15,8 +15,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="image">
+        <a href="index.html">
```

Comment:
The `href` attribute in the anchor tag should not lead to 'index.html' as it is common to use `#` for the home page link in the navigation bar.